MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -17,17 +17,25 @@ final class FeedViewModel {
 	var title: String {
 		Localized.Feed.title
 	}
-
+	
+	var loadError: String {
+		Localized.Feed.loadError
+	}
+	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
-
+	var onFeedLoadError: Observer<Void>?
```

`onFeedLoadError: Observer<Void>?` could be `onErrorStateChange: Observer<String?>?`, so the ViewModel decides when to show/hide errors (instead of the view controller). It's often easier to manage state when all changes are centralized.

For example, in the `loadFeed()` method, the ViewModel can call  `onErrorStateChange(.none)` to hide the error before loading the feed.

And when there's an error, the ViewModel can call `onErrorStateChange(Localized.Feed.loadError)` to show the error.